tasks/json/parser.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * You need to write a parser, which takes the ./test.json file +
+ * and will create a new JSON file with the name parsed.json with the structure:
+ * [
+ *  {
+ *    "docId": "http://doc.epam.com/077b1523-56f0-492a-b954-1269e3acc191"
+ *  },
+ *  {
+ *    "docId": "http://doc.epam.com/077d3a4c-a2fe-4ab9-9046-89c0ce5c5b38"
+ *  }
+ *  .....
+ * ]
+ * where 077b1523-56f0-492a-b954-1269e3acc191 - is the value of the name attribute from the ./test.json file
+ * Please NOTE, that you should omit the .html extension
+ * Please do the task in the scope of the file. Create a pull request a share it with your mentor.
+ */
+const fs = require('fs');
+const objs= require('./test.json');
+
+ 
+const arr = Object.values(objs.list.entries);
```

Comment:
Actually, you can simply use .map() operator:
const arr = objs.list.entries.map(item => {docId: 'http://doc.epam.com/' item.entry.name.replace('.html', '')})';

Map() is more appropriate here since the method is used to change the initial array and as you can see, it can be done in one line :).

Anyway, I'm accepting your task, thank you.